### DISCUSSION TITLE
Add transferrable and locked currency amounts to the Assets Overview section

### DISCUSSION THREAD

**Discussion by polkadirk** [OAK-Foundation/OAK-blockchain#491] [Automation Hub]
The current table shows the following fields;

- Asset
- Available
- Reserved
- Total

However with this information a user is unable to tell the quantity that they have locked in-stake or how much is available to be staked on their wallet. Polkadot.js uses the terms Transferrable and Locked to indicate these values. Screenshots attached below.

![assets overview](https://github.com/OAK-Foundation/OAK-blockchain/assets/104081857/ef5b62a5-eaca-4153-846d-ad4291ae1bf0)
![pjs assets](https://github.com/OAK-Foundation/OAK-blockchain/assets/104081857/5181dd07-9c55-4162-b197-fd1b2a32d561)

**Comment by polkadirk** [comment]
Note - The Available and Total fields in the current system are the same so one can be replaced.

Reserved is also different than locked as reserved is for governance tokens so can keep that field.

**Comment by v9n** [comment]
this is done now
![CleanShot 2024-01-19 at 10 25 25](https://github.com/OAK-Foundation/OAK-blockchain/assets/49754/9420c5ab-011d-401e-aedc-a8782a9a8f2b)
